src/ModelSearchAspect.php

```diff
@@ -118,10 +119,18 @@ protected function addSearchConditions(Builder $query, string $term)
         $attributes = $this->attributes;
         $searchTerms = explode(' ', $term);
 
-        $query->where(function (Builder $query) use ($attributes, $term, $searchTerms) {
+        $grammar = $query->getGrammar();
+
+        // Workaround: Laravel 6 incorrectly returns the builder object instead of the grammar
```

Comment:
Let's just drop support for Laravel 6 in composer.json so we can remove this line.

Reply:
Should I remove it from composer.json or should I just remove the commit and L6 support will be dropped before merging?

Reply:
Feel free to remove it from composer.json

Reply:
Removed it from composer.json and test runner, also removed the old testbench and phpunit versions too.

Reply:
It looks like It was fixed somewhere in the L7 branch since its broken there on the prefer lowest setting. L7 does not receive security fixes anymore, should I remove that too?

Reply:
I dropped it for now, if it's not needed I will forcepush without the latest commit.